packages/hash/frontend/src/components/layout/PageHeader/SearchBar/index.tsx

```diff
@@ -56,6 +64,7 @@ const ResultList: React.FC<{
     component="ul"
     sx={(theme) => ({
       position: !isMobile ? "absolute" : "unset",
+      top: !isMobile ? "calc(100% + 1px)" : "unset",
```

Comment:
Adding a `1px` offset to the `top` value to compensate for the focus width change.